mycity/mycity/intents/food_truck_intent.py

```diff
@@ -19,27 +19,45 @@
 MILE = 1600
 BASE_URL = 'https://services.arcgis.com/sFnw0xNflSi8J0uh/arcgis/rest/' \
                'services/food_trucks_schedule/FeatureServer/0/'
-QUERY = {'where': '1=1', 'out_sr': '4326'}
+
+QUERY = {
+    "distance":"1",
+    "inSR":"4326",
+    "outSR":"4326",
+    "where":"1=1",
```

Comment:
I think we can get rid of this line, since that's the default value. See [query arguments](https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.features.toc.html#arcgis.features.FeatureLayer.query)